portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase

```diff
@@ -500,6 +500,60 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-78055. Liferay Admin user, for the virtual instance, is not able to login after the 'Required' box in LDAP configurations is turned on. ."
+	@priority = 4
+	test LoginWithRequiredBoxTurnedOn {
+		property apacheds.enabled = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given: The main instance and a virtual instance is connected to the same LDAP Server") {
+			var enableSettingList = "enabled,required";
+
+			ApplicationsMenu.gotoPortlet(
+				category = "System",
+				panel = "Control Panel",
+				portlet = "Virtual Instances");
+
+			PortalInstances.addCP(
+				mailDomain = "www.able.com",
+				virtualHost = "www.able.com",
+				webId = "www.able.com");
+
+			LDAP.addLDAPServerAndTestConnection(ldapServerName = "ApacheDS Server");
+
+			User.firstLoginPG(
+				password = "test",
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
+
+			LDAP.addLDAPServerAndTestConnection(
+				baseURL = "http://www.able.com:8080",
+				ldapServerName = "ApacheDS Server");
+		}
+
+		task ("When: 'Enabled' and 'Required' settings are turned on in LDAP settings") {
```

Comment:
Please mention the settings are enabled on both default and virtual instances

Reply:
Fixed.